### ISSUE TITLE
Support for Data Types like Geography Locations Stocks

### ISSUE THREAD

**Issue by SuperDaveOsbourne** [EPPlusSoftware/EPPlus#383]
What is the road map for special internet aware data types like Geography. Locations, and Stocks to be inserted into sheets?

**Comment by JanKallman** [comment]
We have not look into it, so it's not on the roadmap. I'll add it as an enhancement, so we can estimate the effort to implement it.

**Comment by SuperDaveOsbourne** [comment]
Well probably should add Power Query to the list also as that is the base technology that is at the heart of the special data types. 

[https://docs.microsoft.com/en-us/power-query/power-query-what-is-power-query](url)

**Comment by SuperDaveOsbourne** [comment]
Any movement on this? I have  a product but it needs this feature.

**Comment by JanKallman** [comment]
No, we have not looked at this yet. I thing this is a pretty big feature, so it's not planned in the near future.
